src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx

```diff
@@ -12,6 +12,7 @@ export const LoginBox = () => {
   const [userDetails, setUserDetails] = useState({ email: "", password: "" });
   const [error, setError] = useState({ state: false, text: "" });
   const [type, setType] = useState(true);
+  const [trigger, setTrigger] = useState(true)
 
   const validation = /^(?=.*\d)(?=.*[a-z]).{5,10}$/;
```

Comment:
minor refactor you can create a regex object and create a function around it 
then you can use it everywhere instead of defining it in every file you need 
you can also have a set of validation functions at one place that you can use everywhere
```js
export const validationRegex = new Regex('/^(?=.*\d)(?=.*[a-z]).{5,10}$/')
emailRegex.test(email)
```

Reply:
Thanks for a great suggestion! I will check this out